components/prism-scheme.js

```diff
@@ -18,12 +18,12 @@ Prism.languages.scheme = {
 	},
 	'boolean' : /#[tf]/,
 	'operator': {
-		pattern: /(\()(?:[-+*%\/]|[<>]=?|=>?)/,
+		pattern: /(\()(?:[-+*%\/]|[<>]=?|=>?)(?:(?=\s)|$)/,
```

Comment:
Could you please make it `(?=\s|$)`?
It's shorter because the non-capturing group can be removed.